src/components/studies/StudyList.test.tsx

```diff
@@ -0,0 +1,94 @@
+import * as useUserSessionDataState from '@helpers/AuthContext'
```

Comment:
You are testing a small subset of the studyList functionality so you want to wrap it in 'describe' 
_https://www.webtips.dev/webtips/jest/describe-vs-test-vs-it_